content/articles/design-a-sign-up-and-login-gui-using-javafx/index.md

```diff
@@ -0,0 +1,516 @@
+<Label layoutX="130.0" layoutY="515.0" prefHeight="20.0" prefWidth="90.0" style="-fx-text-fill: #A9A9A9;" text="Gender">
+    <font>
+        <Font name="SansSerif Regular" size="15.0" />
+    </font>
+</Label>
+<RadioButton layoutX="300.0" layoutY="515.0" mnemonicParsing="false" selected="true" text="Male" textFill="DARKGRAY">
+    <toggleGroup>
+        <ToggleGroup fx:id="Gender" />
+    </toggleGroup>
+    <font>
+        <Font size="14.0" />
+    </font>
+</RadioButton>
+<RadioButton layoutX="390.0" layoutY="515.0" mnemonicParsing="false" text="Female" textFill="DARKGRAY" toggleGroup="$Gender">
+    <font>
+        <Font size="14.0" />
+    </font>
+</RadioButton>
+<CheckBox fx:id="termsConditionsCheckbox" layoutX="160.0" layoutY="545.0" mnemonicParsing="false" selected="true" style="-fx-text-fill: #A9A9A9;" text="I have read the Terms and Conditions">
+    <font>
+        <Font name="SansSerif Regular" size="15.0" />
+    </font>
+</CheckBox>
+
+<!-- Sign up button -->
+<Button fx:id="signUpButton" layoutX="230.0" layoutY="615.0" mnemonicParsing="false" prefHeight="33.0" prefWidth="130.0" style="-fx-background-color: #24a0ed; -fx-text-fill: white; -fx-border-radius: 5; -fx-border-width: 2; -fx-border-color: #24a0ed;" text="Sign-Up">
+    <font>
+        <Font size="15.0" />
+    </font>
+</Button>
+<Button layoutX="200.0" layoutY="660.0" mnemonicParsing="false" style="-fx-background-color: transparent; -fx-text-fill: #A9A9A9;" text="Terms and Conditions" underline="true">
+    <font>
+        <Font size="15.0" />
+    </font>
+</Button>
+<!-- The End of Signup form -->
+```
+
+##### Footer
+
+In the footer section, you will add your copyright. Drag and drop a label control into the bottom section of your main BorderPane.
+
+This label will have the following Property attributes: `Text`: *Shop © 2021*. Set the following styles: `-fx-text-fill`:  *#B9a9a9*. It will have the following layouts: `Pref Width`: 107, `Pref Height`: 16.
+
+The FXML Code will look as shown below:
+
+```xml
+<!-- Beginning of the bottom part of the Borderpane -->
+<bottom>
+
+    <!-- Simple Copyright -->
+    <Label prefHeight="16.0" prefWidth="107.0" style="-fx-text-fill: #B9a9a9;" text="Shop © 2021" BorderPane.alignment="CENTER" />
+</bottom>
+<!-- Ending of the bottom part of the Borderpane -->
+```
+
+> **NOTE**: Paths may vary depending on your project name and machine. If controls and other elements have been added using code, make sure to import them into the project.
+
+### Run the Application
+
+Run the application as before using the `Shift + F10` shortcut. The results are as shown below:
+
+![Final Product](/engineering-education/design-a-sign-up-and-login-gui-using-javafx/Final-product.png "Final Product")
+
+Find the source code and files in [this](https://github.com/RisoriTofa/A-JavaFX-Login-and-Sign-Up-Form) repository.
+
+### Conclusion
+
+In conclusion, JavaFX is a stable, efficient, and secure UI library for Java developers.
```

Comment:
I would revise this conclusion. It sounds like you are concluding/proving, based on what you did in this guide, that JavaFX is a stable, efficient, and secure UI library. Instead, this guide was to teach users about the basics of the library and how to use it to create a login and signup form. Instead, maybe start off with *In this guide, we...* and go on to list the things learned.

Reply:
:+1:  Understood